site/assets/keedrop.js

```diff
@@ -2,198 +2,200 @@
 ---
 "use strict";
 
-function defaultAdapter(baseUrl) {
-  function request(url, data, callback) {
-    var req = new XMLHttpRequest();
-    req.open(data ? "POST" : "GET", url);
-    req.onreadystatechange = function() {
-      if (4 === req.readyState) {
-        if (req.status < 200 || req.status >= 400) {
-          callback(new Error("Could not send encrypted data to server"), "");
-        } else {
-          callback(undefined, JSON.parse(req.responseText));
+(function(window) {
+  function defaultAdapter(baseUrl) {
+    function request(url, data, callback) {
+      var req = new XMLHttpRequest();
+      req.open(data ? "POST" : "GET", url);
+      req.onreadystatechange = function() {
+        if (4 === req.readyState) {
+          if (req.status < 200 || req.status >= 400) {
+            callback(new Error("Could not send encrypted data to server"), "");
```

Comment:
Left over?

Reply:
No, the error text is never used. When the callback is called it uses its own error (translated) to show the user.